www/source/partials/docs/_using-hab-install-faq.html.md.erb

```diff
@@ -11,21 +11,14 @@ A: We publish compiled packages for OSX, Linux, and Windows. `hab` has a require
 
 **Q: What if I need an old version of `hab`?**
 
-A: We've got you covered! The script we provide for doing curlbash installations will allow you to specify a `-v` flag to pull down a specific version of habitat.
+A: We've got you covered! The script we provide for doing curlbash installations will allow you to specify a `-v` flag to pull down a specific version of Habitat.
 
 **Q: Oh! A curlbash I (love||hate) those.**
 
 A: Indeed they are divisive, we know, that's why we provide a few different ways for you to download. If you'd like to take a look at the script in advance of running it, you can find it in [the core habitat repo](https://github.com/habitat-sh/habitat/blob/master/components/hab/install.sh).
 
-Install Habitat via curlbash using the following command:
-`curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash`
-
-If you're staunchly in the anti-curlbash camp you can get the latest packages from our [download page](https://habitat.sh/downloads) or via releases on [our bintray page](https://bintray.com/habitat)
-
-**Q: Er, how do I install habitat?**
-
-A: You'll need to download the `hab` package. Extract the binary to somewhere on your $PATH (or somewhere that you can execute the binary from a command line). If you're installing on a workstation we would strongly recommend running `hab setup` after youve extracted the binary. Read our [step by step docs here](/docs/using-habitat/#install-habitat).
+If you're staunchly in the anti-curlbash camp, you can get the latest packages from the [download links](/docs/install-habitat/#install-habitat) listed previously, or via releases on our [Bintray page](https://bintray.com/habitat).
 
 **Q: How do I install `hab` across my server fleet?**
 
-A: for the most part, we leave that up to you. You could just use the aforementioned curl-bash with your provisioner of choice. If your app was dockerized with habitat then you won't even need to ask this question, because you'll have everything you need inside your container. We are working on first class mesos dc/os, cloudfoundry and kubernetes integrations - which you can keep up to date on in [our best practices section](/docs/best-practices/) and [blog](/blog).
+A: for the most part, we leave that up to you. You could just use the aforementioned curl-bash with your provisioner of choice. If your app was dockerized with Habitat then you won't even need to ask this question, because you'll have everything you need inside your container. We are working on first class Mesosphere DC/OS, Cloud Foundry, and Kubernetes integrations - which you can keep up to date on in [our best practices section](/docs/best-practices/) and [blog](/blog).
```

Comment:
`For the most...` (cap F)

Sorry if I'm killing you on these. :)

Reply:
You're a great docs person 👍